modules/apps/object/object-service/service.xml

```diff
@@ -225,10 +225,17 @@
 		<finder name="ObjectDefinitionId" return-type="Collection">
 			<finder-column name="objectDefinitionId" />
 		</finder>
+		<finder name="ObjectEntryFolderId" return-type="Collection">
```

Comment:
@AliciaGarciaGarcia, do we use this finder? I haven't seen it in the code (surely it's me) :face_in_clouds:

Reply:
I do not use it, but this way we create the index for the ObjectEntryFolderId on the Entry table. 

I thought a good idea.

No?

Reply:
Better to only create indexes that we are going to use

Reply:
<img src="https://media1.giphy.com/media/9EBjBLa2Idyqk/giphy.gif?cid=bd3ea57e9dhgh9ffnufdlqrjcq1liyaf04jnzf48kg6mzove&ep=v1_gifs_search&rid=giphy.gif&ct=g"/>

Reply:
<img src="https://media2.giphy.com/media/l0ErMlL0QhbgBh4J2/giphy.gif?cid=bd3ea57ewcmpi15tnfmdpci1tsa3x3rrknnbp0b0jjdvo7hl&ep=v1_gifs_search&rid=giphy.gif&ct=g"/>